pkg/rbd/rbd_util.go

```diff
@@ -238,11 +238,11 @@ func rbdManagerTaskDeleteImage(ctx context.Context, pOpts *rbdVolume, cr *util.C
 	if err != nil {
 		if strings.Contains(string(output), rbdTaskRemoveCmdInvalidString1) &&
 			strings.Contains(string(output), rbdTaskRemoveCmdInvalidString2) {
-			klog.Infof(util.Log(ctx, "cluster with cluster ID (%s) does not support Ceph manager based rbd image"+
+			klog.V(3).Infof(util.Log(ctx, "cluster with cluster ID (%s) does not support Ceph manager based rbd image"+
 				" deletion (minimum ceph version required is v14.2.3)"), pOpts.ClusterID)
 			return false, err
 		} else if strings.HasPrefix(string(output), rbdTaskRemoveCmdAccessDeniedMessage) {
-			klog.Infof(util.Log(ctx, "access denied to Ceph MGR-based RBD image deletion "+
+			klog.V(3).Infof(util.Log(ctx, "access denied to Ceph MGR-based RBD image deletion "+
```

Comment:
same here also, use warningf

Reply:
Thank you for reminding me and I push a new commit .